src/credentials/BUILD.gn

```diff
@@ -43,6 +43,10 @@ static_library("credentials") {
     "examples/DefaultDeviceAttestationVerifier.h",
     "examples/DeviceAttestationCredsExample.cpp",
     "examples/DeviceAttestationCredsExample.h",
+    "examples/ExampleDACs.cpp",
+    "examples/ExampleDACs.h",
+    "examples/ExamplePAI.cpp",
+    "examples/ExamplePAI.h",
```

Comment:
We need a new target for these *example values* otherwise everyone has to forcibly use this code, which prevents overriding properly. Could you make a new target with all the `examples/` files and we will do a second PR later to actually make it not included by default.

Reply:
This is not any worse than what we had before. If anything, I would prefer to do this in a follow up PR.

Reply:
Yes, follow-up would work.